tensorflow_datasets/sequence/quickdraw.py

```diff
@@ -0,0 +1,161 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""QuickDraw dataset."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import os
+import numpy as np
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+import tqdm
+
+# Shared constants
+_QUICKDRAW_BASE_URL = (
+    "https://storage.googleapis.com/quickdraw_dataset"
+)  # pylint: disable=line-too-long
+_QUICKDRAW_LABELS_FNAME = "sequence/quickdraw_labels.txt"
+
+_CITATION = """\
+@misc{1704.03477,
+Author = {David Ha and Douglas Eck},
+Title = {A Neural Representation of Sketch Drawings},
+Year = {2017},
+Eprint = {arXiv:1704.03477},
+}
+"""
+
+
+class QuickdrawSketchRNN(tfds.core.GeneratorBasedBuilder):
+    """Quickdraw sequence of strokes dataset used for Sketch RNN.
+
+    This is the version of the QuickDraw data used to train the SketchRNN model.
+    """
+
+    VERSION = tfds.core.Version("1.0.0")
+
+    def _info(self):
+        labels_path = tfds.core.get_tfds_path(_QUICKDRAW_LABELS_FNAME)
```

Comment:
Use 2 spaces indentation instead of 4 [see the guide](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md#5-check-your-code-style)